docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #75.                is_cands = _prune_candidates(T, m, min_dist, is_cands, T_sliding_mu, T_sliding_std)
</pre><p>Replace with</p><p> </p><pre class="ql-syntax" spellcheck="false">is_cands = _prune_candidates(T, m, min_dist, include, is_cands, M_T, Σ_T)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Just noticed something here. I am assuming `include` is a typo. correct? (because the function `_prune_candidates` does not have the input `include`)

Reply:
Yes, apologies as I copy and pasted from _select_candidates

Reply:
Np. I just wanted to see if you have something particular in mind or not.